Src/Support/Google.Apis.Auth.Tests/OAuth2/DefaultCredentialProviderTests.cs

```diff
@@ -254,45 +280,79 @@ public async Task GetDefaultCredential_ExternalAccountCredential_NoCredentialSou
             await Assert.ThrowsAsync<InvalidOperationException>(() => credentialProvider.GetDefaultCredentialAsync());
         }
 
-        [Fact]
-        public async Task GetDefaultCredential_UrlSourcedExternalAccountCredential()
+        public static TheoryData<string, Type> ExternalAccountCredentialTestData
+        {
+            get => new TheoryData<string, Type>
+            {
+                { DummyUrlSourcedExternalAccountCredentialFileContents, typeof(UrlSourcedExternalAccountCredential) },
+                { DummyFileSourcedExternalAccountCredentialFileContents, typeof (FileSourcedExternalAccountCredential) },
```

Comment:
Nit: remove space after `typeof`. (I wouldn't have noticed it if it hadn't been for the inconsistency with the one above.

Reply:
(Note: both of these comments apply twice below as well.)